extensions/doc/makeLuaDocs.lua

```diff
@@ -213,7 +214,7 @@ print("++ Parsing manual...")
                 text             = {}
                 Keys[k].manpos   = posTable
                 Keys[k].mantext  = text
-            else
```

Comment:
If this `else` clause isn't being used, then let's just remove it entirely, rather than leave it commented out.